keps/sig-network/2445-NodeLocal-DNS-cache/kep.yaml

```diff
@@ -0,0 +1,19 @@
+title: NodeLocal DNS Cache
+kep-number: 2445
```

Comment:
This one is tricky, because technically we already have this:
https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/1024-nodelocal-cache-dns

I guess this one should probably be somehow merged with that one KEP.

Reply:
yeah, those 2 should merge, we don't need a whole new KEP

Reply:
@shekhar-rajak  - So let's maybe leave this out of the PR and merge those in a separate PR.

Reply:
Thanks @wojtek-t @thockin for the review. Now "NodeLocal DNS Cache" key is not migrated to new template. I have updated the PR